packages/ast/src/traverse.js

```diff
@@ -2,7 +2,7 @@
 
 type Cb = (type: string, path: NodePath<Node>) => void;
 
-const debug = require("debug")("wasm:traverse");
+const debug = require("debug")("ast:traverse");
```

Comment:
I needed a way to grep for all our debug messages (excluding mocha ones for example), and I decided to pick the wrong `wasm` naming.

Maybe we could add `webassemblyjs:...` as prefix. I'll open an issue and send a PR.